README.md

```diff
@@ -6,7 +6,7 @@ For an introduction of Earthly see the [Earthly GitHub repository](https://githu
 
 ## Installation Notes
 
-Use your favorite plugin manager.
+Use your favorite plugin manager and ensure vim syntax highlighting is turned on.
```

Comment:
I would add this to a different section; If anything I would change this to:

```suggestion
Use your favorite plugin manager:
```

since the following sections apply to common plugin managers.

Reply:
That makes sense. I'll put in a requirements section instead